### PR TITLE
[Docgen] Hide internal, ```Q# → ```qsharp, and fix ms.topic.

### PR DESCRIPTION
**Pull request by cgranade**

### CODE REVIEW

**Review by guenp — APPROVED** [review]

File: src/Documentation/DocumentationGenerator/ProcessDocComments.cs [hunk 1]
```diff
@@ -141,7 +149,10 @@ public override QsCustomType OnTypeDeclaration(QsCustomType type)
                     range: null, // TODO: provide more exact locations once supported by DocParser.
                     source: type.Source.AssemblyOrCodeFile);
 
-                this.writer?.WriteOutput(type, docComment)?.Wait();
+                if (!type.Modifiers.Access.Equals(QsCompiler.SyntaxTokens.AccessModifier.Internal))
```
Comment (samarsha) [inline src/Documentation/DocumentationGenerator/ProcessDocComments.cs:152]: Could you add a `using` for that?

```suggestion
                if (!type.Modifiers.Access.Equals(AccessModifier.Internal))
```

File: src/Documentation/DocumentationGenerator/ProcessDocComments.cs [hunk 2]
```diff
@@ -195,7 +206,10 @@ public override QsCallable OnCallableDeclaration(QsCallable callable)
                     range: null, // TODO: provide more exact locations once supported by DocParser.
                     source: callable.Source.AssemblyOrCodeFile);
 
-                this.writer?.WriteOutput(callable, docComment)?.Wait();
+                if (!callable.Modifiers.Access.Equals(QsCompiler.SyntaxTokens.AccessModifier.Internal))
```
Comment (samarsha) [inline src/Documentation/DocumentationGenerator/ProcessDocComments.cs:209]:
```suggestion
                if (!callable.Modifiers.Access.Equals(AccessModifier.Internal))
```

**Review by samarsha — APPROVED** [review]

**Comment by samarsha** [comment]
By the way, I think the pipelines should automatically run on every push without needing `/azp run` - is that not working for you?

**Comment by cgranade** [comment]
@samarsha: I've had enough cases where  that hasn't worked that I've gotten a bit paranoid, I guess...

**Pull request merged**